### PR TITLE
refactor(ui): update swagger-ui to v4

### PR DESCRIPTION
**Pull request by volkflo**

### CODE REVIEW

**Comment by volkflo** [comment]
Seems to work fine now

**Comment by msavy** [comment]
Rebased onto latest deps

**Comment by msavy** [comment]
image

Backgrounds of samples don't look right. Is this just a CSS thing?

**Comment by msavy** [comment]
Looks like it's caused by prism.js CSS (used also for markdown editor code section).

prism.css#L7
```css
code[class*="language-"],
pre[class*="language-"] {
	color: black;  // <--- this one by the looks of it.
}
```

**Comment by msavy** [comment]
I'm not sure if it was a caching issue as I can't see the issue any more!

**Review by msavy — CHANGES_REQUESTED** [review]
Putting on request changes, but might just be clarifications required.

File: manager/ui/war/plugins/api-manager/ts/apimanPlugin.ts [hunk 1]
```diff
@@ -530,4 +530,4 @@ import "./client/client-metrics.ts"
 import "./client/client-apis.ts"
 import "./client/client.ts"
 import "./client/client-activity.ts"
-import "./sidebar.ts"
\ No newline at end of file
+import "./sidebar.ts"
```
[the remark this answers is not in the document]
Comment (volkflo) [reply manager/ui/war/plugins/api-manager/ts/apimanPlugin.ts:533]: Mhh the view of github here is a bit strange. 
It is just an empty line that was added to the end of the file by Intellij. 
We could remove that I guess.

File: manager/ui/war/package.json [hunk 2]
```diff
@@ -49,19 +49,22 @@
...
     "luxon": "2.3.2",
     "ng-file-upload": "9.1.1",
     "ng-sortable": "1.2.2",
     "ngtemplate-loader": "2.1.0",
     "patternfly": "3.59.4",
     "prismjs": "1.27.0",
+    "process": "0.11.10",
     "select2": "3.4.5",
-    "swagger-ui": "3.52.5",
+    "stream-browserify": "3.0.0",
```
[the remark this answers is not in the document]
Comment (volkflo) [reply manager/ui/war/package.json:66]: That was added for polyfill, as this was suggested during build in the error log.
But I think we do not need that old polyfill stuff, so I removed it again :)

**Review by msavy — COMMENTED** [review]
All good. Thanks!

**Pull request merged**